firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryOverlayMigrationManager.java

```diff
@@ -16,5 +16,6 @@
 
 public class MemoryOverlayMigrationManager implements OverlayMigrationManager {
   @Override
+  // No-op since with memory persistence we always start afresh, nothing needs to be migrated.
```

Comment:
Nit: We usually have this comment inside the method block.

Reply:
Done.